### ISSUE TITLE
BUG: is_union_type return None sometime

### ISSUE THREAD

**Issue by samsja** [ilevkivskyi/typing_inspect#96]
Hey I think I found a bug

```python
from typing_inspect import is_union_type
print(is_union_type(int))
>>> None
```

I believe `is_union_type` should always return a bool, and in this case it should return False

**Comment by ilevkivskyi** [comment]
Yeah. Should be an easy fix, left operand here https://github.com/ilevkivskyi/typing_inspect/blob/master/typing_inspect.py#L232 should be `MaybeUnionType is not None` instead of plain `MaybeUnionType`. PRs are welcome!

**Comment by samsja** [comment]
Hey here is the PR : https://github.com/ilevkivskyi/typing_inspect/pull/97
